src/main/java/ch/uzh/ifi/hase/soprafs24/rest/mapper/DTOMapper.java

```diff
@@ -0,0 +1,20 @@
+package ch.uzh.ifi.hase.soprafs24.rest.mapper;
+
+import ch.uzh.ifi.hase.soprafs24.entity.Session;
+
+import ch.uzh.ifi.hase.soprafs24.rest.dto.SessionCreatedDTO;
+import org.mapstruct.*;
+import org.mapstruct.factory.Mappers;
+
+
+
+
+
+@Mapper
+public interface DTOMapper {
+    DTOMapper INSTANCE = Mappers.getMapper(DTOMapper.class);
+
+    //@Mapping(source = "sessionID", target = "sessionID")
+    //@Mapping(source = "playerID", target = "playerID")
+    SessionCreatedDTO convertEntityToSessionCreatedDTO(Session session);
```

Comment:
That seems weird... doesn't map anything? Needs to map a Player to PlayerDTO. We have a problem here because we need @Patertuck's future code.

Edit: I think its best not to return anything yet until @Patertuck's code is merged. Aka comment it out in the Service, like in the other commented lines.

Reply:
Yeah this was put here for me as a setup where to place my code. So to me personally doesn't matter if it returns something or not, as it is still in construction